crates/blockifier/src/execution/contract_class.rs

```diff
@@ -166,6 +166,24 @@ impl Deref for ContractClassV1 {
     }
 }
 
+impl<'de> Deserialize<'de> for ContractClassV1 {
+    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        // Deserialize into a JSON value
+        let json_value: serde_json::Value = Deserialize::deserialize(deserializer)?;
+
+        // Convert into a JSON string
+        let json_string = serde_json::to_string(&json_value)
+            .map_err(|err| DeserializationError::custom(err.to_string()))?;
+
+        // Use try_from_json_string to deserialize into ContractClassV1
+        ContractClassV1::try_from_json_string(&json_string)
+            .map_err(|err| DeserializationError::custom(err.to_string()))
+    }
+}
```

Comment:
The true issue here is that cairo_vm::program doesn't implement serde.
So we have to go through all this. It's inefficient.

I opened an issue on the cairo-vm: https://github.com/lambdaclass/cairo-vm/issues/1660

I think you should push this one first, and then update in the blockifier once it is merged

Reply:
This issue looks https://github.com/lambdaclass/cairo-vm/issues/1660 like it'll take a long time to be implemented and merged, how can we not be blocked by it?